src/Microsoft.Health.Fhir.Api/Features/Headers/HeaderDictionaryExtensions.cs

```diff
@@ -22,6 +22,16 @@ public static void AddRetryAfterHeaders(this IHeaderDictionary responseHeaders,
         {
             retryAfterTimeSpan ??= TimeSpan.FromSeconds(1); // in case this is missing, provide some value so we that the header is always there
 
+            if (responseHeaders.ContainsKey(KnownHeaders.RetryAfterMilliseconds))
+            {
+                responseHeaders.Remove(KnownHeaders.RetryAfterMilliseconds);
```

Comment:
Should we set this to use whichever value is greater? In case we set a retry somewhere else I feel we shouldn't be lowering it here.

Reply:
Reverted this change